frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue

```diff
@@ -289,6 +299,12 @@ function onExchangeChange(exchange: string) {
       :label="t('exchange_settings.inputs.passphrase')"
     />
 
+    <div v-if="exchange.location === 'coinbase'" class="mt-2">
+      <a href="https://portal.cdp.coinbase.com/access/api" target="_blank" rel="noopener noreferrer">
+    {{ t('exchange_keys_form.generate_keys_link') }}
+      </a>
+    </div>
+
```

Comment:
I guess this part is too generic, if we add this one, we need to add it for all exchanges.
So I think just remove it.

I can fix the frontend issues, if you give me access to push to the repo forked in your account.
Or else you need to fix all those issues, and also fix the lint `pnpm run lint:fix`

Reply:
Removed this block. I also gave you permissions to edit the repo, please let me know if you are able to access it.